app/helpers/_flashes_helper.rb

```diff
@@ -0,0 +1,22 @@
+module FlashesHelper
+
+  ACCEPTED_FLASH_HTML_CLASSES = {
+    success: 'alert alert-success',
+    error: 'alert alert-danger'
+  }  
+
+  def flash_htmlclass_for type
+    ACCEPTED_FLASH_HTML_CLASSES[type.to_sym] || nil
+  end
+
+  def each_accepted_flash
+    flash.each do |fl|
+      yield fl if flash_htmlclass_for fl[0]
+    end
+  end
+
+  def any_accepted_flash?
+    flash.any? {|k,v| flash_htmlclass_for k }
```

Comment:
Minor point: prefer to use underscore for unused block parameters. i.e. `{ |k, _| flash_htmlclass_for k }`.
Even more minor point: prefer consistent padding (or not) of curly braces with spaces.